doc/api/os.md

```diff
@@ -378,9 +378,8 @@ added: v0.3.3
 
 The `os.uptime()` method returns the system uptime in number of seconds.
 
-*Note*: Within Node.js' internals, this number is represented as a `double`.
-However, fractional seconds are not returned and the value can typically be
-treated as an integer.
+*Note*: the return value includes fractions of a second on Windows.
```

Comment:
@refack  Could you repeat the proposed wording in the main thread? Somebody may not look in the outdated discussion.